baseband/vdif/tests/test_vdif.py

```diff
@@ -983,6 +984,40 @@ def test_io_invalid(self, tmpdir):
             vdif.open(tmp_file, 's')
 
 
+def test_sequentialfile(tmpdir):
+    """Tests writing and reading of sequential files.
+
+    These tests focus on reading and writing with templates.
+    """
+
+    # Use sample file as basis of a file sequence.
+    with vdif.open(SAMPLE_FILE, 'rs') as fh:
+        header = fh.header0.copy()
+        data = fh.read()
+        dtime = fh.stop_time - fh.start_time
+    data = np.r_[data, data, data]
+
+    # Create a file sequence using template.
+    template = str(tmpdir.join('f.{file_nr:03d}.vdif'))
+    files = sf.FileNameSequencer(template, {})
+    with vdif.open(files, 'ws', file_size=(16 * header.frame_nbytes),
```

Comment:
Same comment about parentheses. The reason for the confusion is that I now feel I have to stare at the code to check whether it is a tuple being passed in or a single value. Use () only when the equation spans multiple lines.